src/OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed/Configuration/Settings.cs

```diff
@@ -57,6 +54,17 @@ private Settings(IConfigurationSource source)
                 }
             }
 
+            var disabledInstrumentations = source.GetString(ConfigurationKeys.DisabledInstrumentations);
+            if (disabledInstrumentations != null)
+            {
+                foreach (var instrumentation in disabledInstrumentations.Split(separator: ','))
+                {
+                    instrumentations.Remove(instrumentation);
```

Comment:
If not found is likely a typo, since other misconfigurations causes an exception it seems that we should do the same here.

Reply:
Not really as it also can contain IL-rewrite instrumentation(s) that the user does not want to enable.

Reply:
It seems worth a follow-up: we should check if there are no typos on the data entered by the user and perhaps ensure consistency between source and bytecode instrumentation. That said it will be better handled in a separate PR.

Reply:
I think we could dump `OTEL_*` env vars.